index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html><html><head>
-    <title>HackDavis Recruitment</title>
+    <title>HackDavis | January 2019</title>
```

Comment:
Please see the updated README on how to deploy this website. Make these changes on recruitment.html and then use the scripts to render the website onto this, or you could just copy over these changes to that file manually.